TaskLayer/XLSearchTask/PostXLSearchAnalysisTask.cs

```diff
@@ -58,12 +58,6 @@ public MyTaskResults Run(string OutputFolder, List<DbForTask> dbFilenameList, Li
             }
             MyTaskResults.AddTaskSummaryText("Target inter-crosslinks within 1% FDR: " + interCsms.Count(p => p.FdrInfo.QValue <= 0.01 && !p.IsDecoy && !p.BetaPeptide.IsDecoy));
 
-            if (xlSearchParameters.WriteOutputForPercolator)
```

Comment:
I'd like to keep the percolator output.

Reply:
![image](https://user-images.githubusercontent.com/16841846/71118662-27881300-219e-11ea-9519-2e09d6dc0de6.png)